modules/cluster-logging-collector-log-forward-gcp.adoc

```diff
@@ -8,14 +8,14 @@
 
 You can forward logs to link:https://cloud.google.com/logging/docs/basic-concepts[Google Cloud Logging] in addition to, or instead of, the internal default {ocp-product-title} log store.
 
-[NOTE]
+[IMPORTANT]
 ====
-Using this feature with Fluentd is not supported.
+Forwarding logs to GCP is not supported on Red{nbsp}Hat OpenShift on AWS.
```

Comment:
@cahartma is this true?  is there a hard restriction?  Seems unlikely someone would do this but not certain it is not possible

Reply:
Yes, GCP is one of our outputs.  The logging portion of GCP (and our output name) is `googleCloudLogging`

Reply:
So, an openshift cluster hosted on AWS works fine.   However, if you are running a ROSA cluster, maybe there is a support exception?   I'd think they can forward off-cluster (to GCP) without the requirement of tokens, as they would with any other non-cloudwatch ouput?